htdocs/Language/default.php

```diff
@@ -217,7 +217,9 @@
 		"HEIGHT" => "Height",
 		"FONT_SIZE" => "Font Size",
 		"COL_COUNT" => "Column-Count",
-		"CMD_REFRESH" => "Refresh"
+		"CMD_REFRESH" => "Refresh",
+		"SPECIMEN_CUSTOM_FIELDS" => "Specimen Custom Fields",
+		"PATIENT_CUSTOM_FIELDS" => "Patient Custom Fields"
```

Comment:
You did this correctly in the PHP files, but please also update `default.xml` and `en.xml` as well.